packages/styles/src/utilities/_variables.scss

```diff
@@ -31,6 +31,144 @@ $position-values: (
 );
 
 $utilities: (
+  'border': (
+    responsive: true,
+    property: border,
+    class: border,
+    values: (
+      null: var(--post-border-width) var(--post-border-style)
+        rgba(var(--post-border-color-base), var(--post-border-opacity)),
+      0: 0,
+    ),
+  ),
+  'border-top': (
+    property: border-top,
+    values: (
+      null: var(--post-border-width) var(--post-border-style) var(--post-border-color),
+      0: 0,
+    ),
+  ),
+  'border-end': (
+    property: border-right,
+    class: border-end,
+    values: (
+      null: var(--post-border-width) var(--post-border-style) var(--post-border-color),
+      0: 0,
+    ),
+  ),
+  'border-bottom': (
+    property: border-bottom,
+    values: (
+      null: var(--post-border-width) var(--post-border-style) var(--post-border-color),
+      0: 0,
+    ),
+  ),
+  'border-start': (
+    property: border-left,
+    class: border-start,
+    values: (
+      null: var(--post-border-width) var(--post-border-style) var(--post-border-color),
+      0: 0,
+    ),
+  ),
+  'border-width': (
+    responsive: true,
```

Comment:
The border utilities probably do not need to be responsive, there is no real use case for this and this adds a lot of classes.
```suggestion
```